src/ExpressionEvaluator/Core/Source/ResultProvider/ResultProvider.cs

```diff
@@ -466,20 +576,22 @@ private static string GetTypeName(DkmInspectionContext inspectionContext, DkmClr
             DkmClrCustomTypeInfo declaredTypeInfo,
             DkmInspectionContext inspectionContext,
             string name,
+            string fullName,
+            ReadOnlyCollection<string> formatSpecifiers,
             CompletionRoutine<DkmEvaluationResult> completionRoutine)
         {
-            var type = value.Type.GetLmrType();
-            if (type.IsTypeVariables())
+            var type = value.Type;
```

Comment:
TypeAndCustomInfo at line 587 requires a DkmClrType now. Reverted this change and passed declaredType to TypeAndCustomInfo instead.